src/pages/en/guides/images.md

```diff
@@ -142,7 +141,9 @@ You must also either provide `width` and `height`, or one of the dimensions plus
 
 #### Local Images in `public/`
 
-The `<Image />` component can also be used with images stored in the `public/` directory and the `src` attribute is relative to the public folder. It will be treated as a remote image, which requires either both `width` and `height`, or one dimension and an `aspectRatio` attribute.
+The `<Image />` component can also be used with images stored in the `public/` directory and the `src` attribute is relative to the public folder. It will be treated as a remote image, which requires either both `width` and `height`, or one dimension and an `aspectRatio` attribute. 
```

Comment:
Ooh, this is weird! 😂 TIL.

Reply:
This is not the weirdest thing about our image integration... 😅